### PR TITLE
[doc] Fix scala doc in javadsl.FileIO

### PR DESCRIPTION
**Pull request by tisonkun**
Straightforward. Links don't work without the patch.

### CODE REVIEW

File: akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala [hunk 1]
```diff
@@ -9,7 +9,7 @@ import java.nio.file.{ OpenOption, Path }
 import java.util
 import java.util.concurrent.CompletionStage
 
-import akka.stream.{ javadsl, scaladsl, IOResult }
+import akka.stream.{ javadsl, scaladsl, ActorAttributes, IOResult }
```
Comment (johanandren) [inline akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: Why is this import needed though?

Comment (TisonKun) [reply akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: Otherwise the link `[[ActorAttributes]]` in scala doc below doesn't work. We can, instead, use `akka.stream.ActorAttributes` below. Both way is OK for me.

**Comment by raboof** [comment]
OK TO TEST

File: akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala (hunk 1 above)
Comment (raboof) [reply akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: I'm afraid it'll need to be `akka.stream.ActorAttributes`, as it seems the compiler produces an 'unused import' error when it is only used in scaladoc(?)

Comment (johanandren) [reply akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: I was under the impression that importing it into scope just takes a red-squiggly in IntelliJ away but it still won't render in the scaladoc unless it is the fully qualified name.

Comment (TisonKun) [reply akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: Thanks for your information, updating...

Comment (TisonKun) [reply akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala]: Updated.

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Comment by TisonKun** [comment]
Thanks for your review @johanandren ! Could you take a look at this @raboof ? I think it is already mergeable.

**Comment by raboof** [comment]
Tested with `sbt -Dakka.genjavadoc.enabled=true unidoc` and looking at `target/scala-2.12/unidoc/akka/stream/javadsl/FileIO$.html` and `target/javaunidoc/akka/stream/javadsl/FileIO.html`. Unfortunately they don't look quite right yet to me:

DeepinScreenshot_select-area_20191209173103
DeepinScreenshot_select-area_20191209173132

https://github.com/akka/akka/blob/master/CONTRIBUTING.md#scaladoc has some notes though I'm not sure they're exactly relevant to this particular reference.

**Comment by TisonKun** [comment]
Thanks for your information @raboof . Sorry I miss the notification for a bit. I trying to follow the document and replace `[[toPath(Path, util.Set[OpenOption])]]` with `[[FileIO.toPath(Path, util.Set[OpenOption])]]`. However, it was rendered as `<span class="extype" name="FileIO.toPath(Path,">util.Set[OpenOption])</span>` and so does `[[#toPath(Path, util.Set[OpenOption])]].

I investigate a bit and find that genjavadoc only distinguish `[[FileIO.toPath]]` & `[[toPath]]` that is no arguments. Maybe it is a shortcoming of genjavadoc and in this pull request let's first make it work in IDEA.

see also https://github.com/lightbend/genjavadoc/issues/43#issuecomment-565765397

**Review by raboof — APPROVED** [review]
You're right that it current doesn't work either, so making it work in IDEA is an improvement here.

The next step might be to make this link 'valid scaladoc', but since it seems liking to methods in 'valid scaladoc' is pretty horrible (https://stackoverflow.com/questions/15394322/how-to-disambiguate-links-to-methods-in-scaladoc) perhaps we should first propose new features to scaladoc linking :D. After that we could look into correctly converting those links in genjavadoc.

**Comment by TisonKun** [comment]
Thanks for your reply @raboof ! Any other concern in this PR? I think it is mergeable now.
